test/test_mpas.py

```diff
@@ -30,13 +32,16 @@ def test_read_mpas(self):
     def test_mpas_to_grid(self):
         """Tests creation of Grid object from converted MPAS dataset."""
         mpas_uxgrid_primal = ux.open_grid(self.mpas_grid_path, use_dual=False)
-        mpas_uxgrid_primal = ux.open_grid(self.mpas_grid_path, use_dual=True)
+        mpas_uxgrid_dual = ux.open_grid(self.mpas_grid_path, use_dual=True)
+        pass
 
     def test_primal_to_ugrid_conversion(self):
         """Verifies that the Primal-Mesh was converted properly."""
 
-        # primal-mesh encoded in the UGRID conventions
-        ds, _ = _read_mpas(self.mpas_xr_ds, use_dual=False)
+        for path in [self.mpas_grid_path, self.mpas_ocean_mesh]:
+            # dual-mesh encoded in the UGRID conventions
+            uxgrid = ux.open_grid(path, use_dual=False)
+            ds = uxgrid._ds
```

Comment:
I'm pretty sure the whole content of this function from here needs to be inside the for loop